File: CampingNeretva/UI/campingneretva_desktop/lib/models/activity_comment_model.dart
class ActivityComment {
  final int activityCommentId;
  final int activityId;
  final int userId;
  final String commentText;
  final int rating;
  final DateTime datePosted;

  ActivityComment({
    required this.activityCommentId,
    required this.activityId,
    required this.userId,
    required this.commentText,
    required this.rating,
    required this.datePosted,
  });

  factory ActivityComment.fromJson(Map<String, dynamic> json) {
    return ActivityComment(
      activityCommentId: json['activityCommentId'],
      activityId: json['activityId'],
      userId: json['userId'],
      commentText: json['commentText'],
      rating: json['rating'],
      datePosted: DateTime.parse(json['datePosted']),
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'activityId': activityId,
      'userId': userId,
      'commentText': commentText,
      'rating': rating,
    };
  }
}

File: CampingNeretva/CampingNeretva.Model/Models/ActivityCommentModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model.Models
{
    public class ActivityCommentModel
    {
        public int ActivityCommentId { get; set; }
        public int ActivityId { get; set; }
        public int UserId { get; set; }
        public string CommentText { get; set; }
        public int Rating { get; set; }
        public DateTime DatePosted { get; set; }
    }
}
